bumble/host.py

```diff
@@ -721,6 +721,14 @@ def on_hci_le_advertising_report_event(self, event):
     def on_hci_le_extended_advertising_report_event(self, event):
         self.on_hci_le_advertising_report_event(event)
 
+    def on_hci_le_advertising_set_terminated_event(self, event):
+        self.emit(
+            'advertising_set_terminated',
```

Comment:
To be consistent, the pattern is that event names are nouns, following the same convention as for Web events, so for example we use 'disconnection' rather than 'disconnected', so here this would be 'advertisting_set_termination'.

Reply:
Done.